CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unrealesed
+
+### Added
+
+- A field of type `signatureUpload` has been added, for the `QasFormGenerator` to generate the field automatically.
```

Comment:
Added new type `signatureUpload` to `QasField`. Mas uma duvida, isso seria `camelCase`?

Reply:
não existe nenhum componente com nome composto. O que acha? camelCase ou não?